dell-csi-helm-installer/verify-csi-isilon.sh

```diff
@@ -11,8 +11,8 @@
 #
 # verify-csi-isilon method
 function verify-csi-isilon() {
-  verify_k8s_versions "1.19" "1.22"
-  verify_openshift_versions "4.6" "4.8"
+  verify_k8s_versions "1.20" "1.22"
```

Comment:
i guess this was common check to validated correct ?

Reply:
This is a driver-specific check; we don't have it as a common check. May be in the future as you are suggesting here, we can have a common check as all the drivers are in line with one another with respect to the support matrix.

Reply:
Agree that it should be made common in the future as we should not be diverging.